src/dbcp/validation/tests.py

```diff
@@ -243,7 +243,7 @@ def test_county_wide_coverage(engine: Engine):
         df.shape[0] == n_counties
     ), "counties_wide_format does not contain all counties"
     notnull = df.notnull()
-    n_expected_counties = 2472
+    n_expected_counties = 2459
```

Comment:
The counties that no longer have proposed capacity are in PJM and ISONE which is expected based on the analysis in the quarterly update notebook.